<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SourceInstaller Version 0.1A</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="AutomaticTab">
       <attribute name="title">
        <string>Automatic</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Source file:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dirTextAuto"/>
           </item>
           <item>
            <widget class="QPushButton" name="browseButtonAuto">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="autoCompileButton">
          <property name="text">
           <string>Compile</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="manualTab">
       <attribute name="title">
        <string>Manual</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Source file:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dirTextManual"/>
           </item>
           <item>
            <widget class="QPushButton" name="browseButtonManual">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="sourceTypeGroupbox">
          <property name="title">
           <string>Type of source code</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="configSource">
             <property name="text">
              <string>Regular source code (use ./configure)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="qtSource">
             <property name="text">
              <string>QT project (use qmake)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="autogenSource">
             <property name="text">
              <string>other type of source code (use autogen.sh)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="manualCompileButton">
          <property name="text">
           <string>Compile</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="advancedTab">
       <attribute name="title">
        <string>Advanced</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Source file:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dirTextAdvanced"/>
           </item>
           <item>
            <widget class="QPushButton" name="browseButtonAdvanced">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="sourceTypeGroupbox_2">
          <property name="title">
           <string>Type of source code</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="configSource_2">
             <property name="text">
              <string>Regular source code (use configure file)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="qtSource_2">
             <property name="text">
              <string>QT project (use qmake)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="autogenSource_2">
             <property name="text">
              <string>Another type of source code (use autogen.sh)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="otherSource">
             <property name="text">
              <string>Other command...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="customCompileCommand">
             <property name="text">
              <string>./configure</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="flags_VARS_Groupbox">
          <property name="title">
           <string>Custom flags</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Make flags:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="makeFlags"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Environmental Variables (seperate with a semicolon):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ENV_VARS"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="advancedCompileButton">
          <property name="text">
           <string>Compile</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <action name="actionChoose_file">
   <property name="text">
    <string>Choose a different file...</string>
   </property>
  </action>
  <action name="actionAutomatic_mode">
   <property name="text">
    <string>Automatic mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
